pubtools/pulplib/_impl/client/client.py

```diff
@@ -181,6 +181,15 @@ def search_repository(self, criteria=None):
             response_f, lambda data: self._handle_page(Repository, search, data)
         )
 
+    def get_maintenance_status(self):
+        """Get maintenance status from maintenance repository
```

Comment:
The docs here should have a bit of explanation about what "maintenance status" is.

Also, you don't need to say it's from "maintenance repository" as this should be just an implementation detail, ideally hidden.